monolith/templates/navbar/admin.html

```diff
@@ -1,13 +1,11 @@
-<div class="navbar-collapse collapse w-100 order-1 order-md-0 dual-collapse2">
-    <ul class="navbar-nav mr-auto">
-        <li class="nav-item">
-            <a class="nav-link" href="/">Home</a>
-        </li>
-        <li class="nav-item">
-            <a class="nav-link" href="/list_users">Users</a>
-        </li>
-        <li class="nav-item">
-            <a class="nav-link" href="/report_positive">Report positive</a>
-        </li>
-    </ul>
-</div>
\ No newline at end of file
+
+  <li class="nav-item">
+    <a class="nav-link" href="#">Home
```

Comment:
maybe is better to have an href="/"?

Reply:
yeah, totally agree, it doesn't make sense the "#"

Reply:
Thanks, marked as resolved :-)